python/xorbits/_mars/dataframe/datasource/read_parquet.py

```diff
@@ -665,6 +665,11 @@ def read_parquet(
     is_partitioned = False
     if use_arrow_dtype is None:
         use_arrow_dtype = options.dataframe.use_arrow_dtype
+    if use_arrow_dtype and engine_type != "pyarrow":
+        raise ValueError(
```

Comment:
Maybe raise NotImplementedError?

Reply:
Will not support this kind of behavior in the future, so `NotImplementedError` is not proper.